ansible/roles/k8s_cluster/tasks/install.yml

```diff
@@ -58,6 +58,10 @@
 - include_tasks: docker.yml
   when: install_docker | bool
```

Comment:
Do we still have to install docker when we use containerd ? If yes, then we will have some conflicts with your PR I think so (packages, ...)

Reply:
the idea is to be able to choose which container service to use, since docker is being deprecated but hasn't been removed.